src/patternfly/components/Content/content.scss

```
@@ -82,7 +82,7 @@
   --#{$content}--blockquote--BorderInlineStartWidth: var(--pf-t--global--border--width--300); // TODO semantic value?
 
   // Lists
-  --#{$content}--list--Gap: var(--pf-t--global--spacer--sm);
+  --#{$content}--list--Gap: var(--pf-t--global--spacer--gap--group--vertical);
```

Chatting with @lboehling we may look to add some content specific tokens in the future, so to be consistent with other content elements (description list excluded) let's update this to be the generic t-shirt spacer for now.

```suggestion
  --#{$content}--list--Gap: var(--pf-t--global--spacer--sm);
```